postgres/src/main/scala/zio/sql/postgresql/PostgresModule.scala

```diff
@@ -1,5 +1,205 @@
 package zio.sql.postgresql
 
-import zio.sql.{ Jdbc, Sql }
+import zio.sql.Jdbc
 
-trait PostgresModule extends Sql with Jdbc {}
+trait PostgresModule extends Jdbc { self =>
+
+  override def renderRead(read: self.Read[_]): String = {
```

Comment:
I don't really like having so much code duplicated across the different modules, but I'm also unsure how we might fix this given the path dependent types

Reply:
It might look like a code duplication now, but these renderers will diverge more and more over time I think. Too many differences between DBs even at this level.

Reply:
Very true. At least they all start with "select" :)